NameGame/NameGame/UserRecord.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UserRecord.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+/** 
+    Models an error that's thrown if key can't be parsed from JSON dictionary.
+ */
+enum ParsingError: Error {
+    case missingKey(String)
+}
+
+/** 
+    Models a user record.
+ */
+
+struct UserRecord {
+    
+    let id: String
+    let firstName: String
+    let lastName: String
+    let imageUrl: String
```

Comment:
I think there could be value in modeling out the `Headshot` instead of just extracting the image url.